app/lib/project/MetapolatorProject.js

```diff
@@ -613,28 +612,23 @@ define([
 
         glyphSet = GlyphSet.factory(false, io, dirName+'/glyphs', undefined, 2);
 
-        exportController = new ExportController(master, model, glyphSet, precision);
+        exportController = new ExportController(instance, model, glyphSet, precision);
```

Comment:
I think I made my point, just don't call "masters" "instance" :-) I won't comment further on this in this file.